cmd/vicadmin/logs.go

```diff
@@ -154,15 +154,12 @@ func configureReaders() map[string]entryReader {
 		readers[path[1:]] = fileReader(path)
 	}
 
-	if config.vmPath == "" {
-		log.Info("vm-path not set, skipping datastore log collection")
-	} else {
-		err := findDatastore()
+	log.Info("vm-path not set, skipping datastore log collection")
+	// err := findDatastore()
```

Comment:
Okay, I don't see a difference in the logs either I was just wondering what your thought process was in commenting versus removing it, thanks!